### DISCUSSION TITLE
Running cloze study in cognition.run

### DISCUSSION THREAD

**Discussion by aristia** [jspsych/jsPsych#2885] [Q&A] [answered]
Hi, 
I am trying to run a cloze study using jspsych in cognition. 
I think it's awesome as it is pretty easy to use for a beginner, my question is how can I save the text stimuli as well? since I notice that the saved data is only collect the response but not the text. I am using the code like in the pluginn example 

 var cloze_trial = {
        type: jsPsychCloze,
        text: 'The %% is the largest terrestrial mammal. It lives in both %% and %%.'
    }; 

Thanks in advance

**Comment by jodeleeuw** [comment, marked as answer]
Hi @aristia,

You can use the [save trial parameters](https://www.jspsych.org/7.3/overview/plugins/#the-save_trial_parameters-parameter) option to save the text string.

**Reply by aristia** [reply]
It works perfectly !! 
thanks a lot !!
